Innohoot/DataLayer/Services/Implementations/OptionService.cs

```diff
@@ -1,14 +1,51 @@
-﻿using Innohoot.Models.Activity;
+﻿using AutoMapper;
+using Innohoot.DTO;
+using Innohoot.Models.Activity;
 using Microsoft.EntityFrameworkCore;
 
 namespace Innohoot.DataLayer.Services.Implementations
 {
-	public class OptionService:CRUDService<Option>, IOptionService
+	public class OptionService:IOptionService
```

Comment:
Remove all inherits from CRUDService. With Adding DTO we do need common operation for every service